jdaviz/core/template_mixin.py

```diff
@@ -3315,6 +3327,11 @@ def _update_mixed_state(self):
             for state in self.linked_states:
                 current_glue_values.append(self._get_glue_value(state))
             mixed = self.is_mixed(current_glue_values)
+            # ensure the value corresponds to the first entry, this prevents the case where a glue
+            # change to one of the linked_states changes the value that will be adopted when
+            # unmixing something in mixed state and results in more consistent and predictable
+            # behavior
+            self.value = current_glue_values[0]
```

Comment:
I suspect this change is causing the test failures - will need to think how to do this in a way that guarantees the correct type and avoids recursion.